ingestion/src/metadata/ingestion/source/database/bigquery_usage.py

```diff
@@ -126,10 +126,7 @@ def _get_raw_extract_iter(self) -> Optional[Iterable[Dict[str, Any]]]:
                 f"Scanning query logs for {(self.start+timedelta(days=i)).date()} - {(self.start+timedelta(days=i+1)).date()}"
             )
             entries = self.usage_logger.list_entries(filter_=filter_)
-            yield TableQueries(
-                queries=self.get_table_query(entries),
-                analysisDate=self.start + timedelta(days=i),
-            )
+            yield TableQueries(queries=self.get_table_query(entries))
```

Comment:
@ulixius9 is this correct? In `usage_source` we have `yield TableQueries(queries=query_list, analysis_date=datetime.utcnow())`

Reply:
Ohh right, so that is for running usage through query log file, didn't tested it after schema changes
@OnkarVO7  can you please remove date from [here](https://github.com/open-metadata/OpenMetadata/blob/ea35d77b52e260d00a4858eb2593d4ad7355413f/ingestion/src/metadata/ingestion/source/database/usage_source.py#L104) as well